.github/workflows/snyk-lts.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will scan and monitor all the LTS releases for vulnerabilities with Snyk 
+name: Snyk Container for LTS Releases
+on:
+  push:
+    # These branches represent the LTS releases
+    branches:
+      - 0.12.x
+      - 0.11.x
```

Comment:
Should be `0.12.lts` and `0.11.lts` to match what the document prescribes

Reply:
Wasn't sure if we decided to change the branch names or not. We'll have to update them in the repo as well.